ovos_cli_client/__main__.py

```diff
@@ -12,79 +12,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import curses
+
 import io
-import os.path
-import signal
 import sys
-from os.path import exists
-from ovos_config import Configuration
-from ovos_utils.signal import get_ipc_directory
-from ovos_utils.log import LOG
-from ovos_cli_client.text_client import (
-    load_settings, save_settings, simple_cli, gui_main,
-    start_log_monitor, start_mic_monitor, connect_to_mycroft,
-    ctrl_c_handler
-)
-from ovos_config.meta import get_xdg_base
-from ovos_utils.xdg_utils import xdg_state_home
-
-sys.stdout = io.StringIO()
-sys.stderr = io.StringIO()
-
-
-def custom_except_hook(exctype, value, traceback):
-    print(sys.stdout.getvalue(), file=sys.__stdout__)
-    print(sys.stderr.getvalue(), file=sys.__stderr__)
-    sys.stdout, sys.stderr = sys.__stdout__, sys.__stderr__
-    sys.__excepthook__(exctype, value, traceback)
-
-
-sys.excepthook = custom_except_hook  # noqa
+from mycroft_bus_client import MessageBusClient
 
+from ovos_cli_client.curses_client import launch_curses_tui
+from ovos_cli_client.text_client import launch_simple_cli
 
-def main():
-    # Monitor system logs
-    config = Configuration()
 
-    legacy_path = "/var/log/mycroft"
+def main(simple):
```

Comment:
This breaks python-exec. It expects a main() function without any arguments.

```
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.10/ovos-cli-client", line 8, in <module>
    sys.exit(main())
TypeError: main() missing 1 required positional argument: 'simple'
```

I guess the simple solution is to move the check into the main function.